src/main/java/com/garrytrue/repository/CRUDRepository.java

```diff
@@ -0,0 +1,15 @@
+package com.garrytrue.repository;
+
+/**
+ * Created by garrytrue on 12.03.17.
+ */
+public interface CRUDRepository<T> {
```

Comment:
you can add void deleteAll() method.

it will clear table, list or hashmap.
would be useful in @After tests